packages/instrument-utils/src/storedInstrument/ManageBraintreeAchInstrumentsTable/ManageBraintreeInstrumentsTable.tsx

```diff
@@ -0,0 +1,92 @@
+    onDeleteInstrument(id: string): void;
+}
+
+const ManageInstrumentsRow: FunctionComponent<ManageInstrumentsRowProps> = ({
+    instrument,
+    onDeleteInstrument,
+}) => {
+    const handleDelete = useCallback(() => {
+        onDeleteInstrument(instrument.bigpayToken);
+    }, [instrument, onDeleteInstrument]);
+
+    return (
+        <tr>
+            <td data-test="manage-instrument-bankType">
+                <IconBraintreeAch size={IconSize.Medium} />
+            </td>
+            <td data-test="manage-instrument-accountNumber">{instrument.accountNumber}</td>
+            <td data-test="manage-instrument-issuer">{instrument.issuer}</td>
+            <td>
+                <button
+                    className="button button--tiny table-actionButton optimizedCheckout-buttonSecondary"
+                    data-test="manage-instrument-delete-button"
+                    onClick={handleDelete}
+                    type="button"
+                >
+                    <TranslatedString id="common.delete_action" />
+                </button>
+            </td>
+        </tr>
+    );
+};
+
+export interface ManageBraintreeInstrumentsTableProps {
+    instruments: BraintreeAchInstrument[];
+    isDeletingInstrument: boolean;
+    onDeleteInstrument(id: string): void;
+}
+
+const ManageBraintreeInstrumentsTable: FunctionComponent<ManageBraintreeInstrumentsTableProps> = ({
+    instruments,
+    isDeletingInstrument,
+    onDeleteInstrument,
+}) => {
+    if (instruments.length === 0) {
+        return (
+            <p>
+                <TranslatedString id="payment.instrument_manage_modal_empty_text" />
+            </p>
+        );
+    }
+
+    return (
+        <LoadingOverlay isLoading={isDeletingInstrument}>
+            <table className="table">
+                <thead className="table-thead">
+                    <tr>
+                        <th>
+                            <TranslatedString id="payment.instrument_manage_table_header_payment_method_text" />
+                        </th>
+                        <th>
+                            <TranslatedString id="payment.instrument_manage_table_header_ending_in_text" />
+                        </th>
+                        <th>
+                            <TranslatedString id="payment.instrument_manage_table_header_routing_number_text" />
+                        </th>
+                        <th />
+                    </tr>
+                </thead>
+
+                <tbody className="table-tbody">
+                    {instruments.map((instrument) => (
+                        <ManageInstrumentsRow
+                            instrument={instrument}
+                            key={instrument.bigpayToken}
+                            onDeleteInstrument={onDeleteInstrument}
+                        />
+                    ))}
+                </tbody>
+            </table>
+        </LoadingOverlay>
+    );
+};
+
+export default memo(ManageBraintreeInstrumentsTable);
```

Comment:
Why do we need `memo` here?

Reply:
If I'm not mistaken, memo optimisation is redundant here.
ManageBraintreeInstrumentsTable mounts inside the [ManageInstrumentsModal.tsx](https://github.com/bigcommerce/checkout-js/pull/1289/files/887c284c7010a58ff3450d372b3dd925d7a9609a#diff-b3a71b05a87599ac48968e85e03998fbb089e86607ca3b7adde4e81dc95abba3)
and there is no additional re-renders of the ManageInstrumentsModal so I think we could avoid usage memo in such cases.
cc @bc-peng @animesh1987